src/service/viewer-impl.js

```diff
@@ -221,6 +221,10 @@ export class Viewer {
     this.performanceTracking_ = this.params_['csi'] === '1';
     dev.fine(TAG_, '- performanceTracking:', this.performanceTracking_);
 
+    this.shareTrackingIncomingFragment_ = this.params_['share-tracking'];
```

Comment:
`getParam` can be used if the url has the share-tracking fragment. But the problem is the url cannot has the share-tracking fragment. So we'll use another way to get it. I'll not read from the url params anymore.